Kzl 241 document controller (#86)
kuzzleio/documentation-V2: src/sdk-reference/cpp/1/document/mCreate/index.md

---
layout: sdk.html.hbs
algolia: true
title: mCreate
description:
order: 200
---

# mCreate

> Creates new documents in the persistent data storage.

Reviewer: ```suggestion
Creates multiple documents.
```

> Returns a partial error (with status 206) if one or more documents fail to create.

Reviewer: ```suggestion
Throws a partial error (error code 206) if one or more document creations fail. 
```
